webapp/locales/en.json

```diff
@@ -329,6 +329,28 @@
       "mute": "Mute user",
       "unmute": "Unmute user",
       "unmuted": "{name} is unmuted again"
+    },
+    "blocked-users": {
+      "name": "Blocked users",
+      "explanation": {
+        "intro": "If another user has been blocked by you, this is what happens:",
+        "your-perspective": "You will no longer be able to interact with their contributions.",
+        "their-perspective": "Vice versa: The blocked person will also no longer be able to interact with your contributions.",
+        "notifications": "Blocked users will no longer receive notifications if they mention each other.",
+        "closing": "This should be sufficient for now so that blocked users can no longer bother you.",
+        "commenting-disabled": "Commenting is not possible at this time on this post.",
+        "commenting-explanation": "This can happen for several reasons, please see our "
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-23T17:24:26Z" title="Thursday, January 23rd 2020, 6:24:26 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -329,6 +329,28 @@
       "mute": "Mute user",
       "unmute": "Unmute user",
       "unmuted": "{name} is unmuted again"
+    },
+    "blocked-users": {
+      "name": "Blocked users",
+      "explanation": {
+        "intro": "If another user has been blocked by you, this is what happens:",
+        "your-perspective": "You will no longer be able to interact with their contributions.",
+        "their-perspective": "Vice versa: The blocked person will also no longer be able to interact with your contributions.",
+        "notifications": "Blocked users will no longer receive notifications if they mention each other.",
+        "closing": "This should be sufficient for now so that blocked users can no longer bother you.",
+        "commenting-disabled": "Commenting is not possible at this time on this post.",
+        "commenting-explanation": "This can happen for several reasons, please see our "
```


Translates to: "Dafür kann es mehrere Gründe geben, bitte schau in unsere "